### ISSUE TITLE
Natural language intermediate steps for GSM8K Aug. 

### ISSUE THREAD

**Issue by enkeejunior1** [da03/implicit_chain_of_thought#18]
Hello, 

I am currently try to reproduce the explicit CoT result from your paper, but realized that I need natural language intermediate steps for training GPT-2 for explicit CoT.

Do you have any plans to release the data for training explicit CoT? I would be very grateful if you could. Thank you in advance.

**Comment by chadlzx** [comment]
Hello,
I have also attempted to reproduce the explicit CoT results following the hyperparameters reported in the paper, but I was only able to achieve a validation loss of around 1.0 and a test accuracy of approximately 8%. I can't imagine how to achieve around 40% test accuracy using GPT-2 small.
Could you please share any suggestions or potential areas I might be overlooking in my implementation?
Thank you in advance for your guidance!
